### PR TITLE
[Method References 2] LPS-170883 Convert method reference to ServiceTrackerList in module portal-security-audit-router

### PR DESCRIPTION
**Pull request by W2jx**

### CODE REVIEW

**Comment by W2jx** [comment]
no conflicts(2022/12/13 9:36)

**Comment by zhangyan0701** [comment]
No regressions found.

**Comment by W2jx** [comment]
no conflicts(2022/12/14 9:27)

**Comment by W2jx** [comment]
no conflicts(12/15 9:21)

**Comment by W2jx** [comment]
no conflicts(2022/12/16 8:57)

**Comment by W2jx** [comment]
no conflicts(12/19 9:52)

File: modules/apps/portal-security-audit/portal-security-audit-router/src/main/java/com/liferay/portal/security/audit/router/internal/DefaultAuditRouter.java [hunk 1]
```diff
@@ -121,54 +119,15 @@ protected void modified(Map<String, Object> properties) {
...
-			if (auditMessageProcessorsSet == null) {
-				auditMessageProcessorsSet = new HashSet<>();
-
-				_auditMessageProcessors.put(
-					eventType, auditMessageProcessorsSet);
-			}
-
-			auditMessageProcessorsSet.add(auditMessageProcessor);
-		}
+		_globalAuditMessageProcessors.add(auditMessageProcessor);
```
Comment (dantewang) [inline modules/apps/portal-security-audit/portal-security-audit-router/src/main/java/com/liferay/portal/security/audit/router/internal/DefaultAuditRouter.java]: @W2jx Here `*` means a wildcard match, so the logic adds the one with `*` to the `global` collection. If you want to eliminate the need to divide the processor into global & non-global, you need to first investigate where and how `eventType` is used. Do they have any logic to get with any specific event type? You need to update that part too.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by dantewang** [comment]
@W2jx Closing, please resend after investigating and making corresponding changes, thanks!

**Pull request closed without merging**